packages/repocop/src/evaluation/repository.ts

```diff
@@ -460,6 +460,7 @@ export function snykAlertToRepocopVulnerability(
 	issue: SnykIssue,
 	projects: SnykProject[],
 ): RepocopVulnerability {
+	// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion -- to allow ts version bump
```

Comment:
```suggestion
	// eslint-disable-next-line @typescript-eslint/no-unnecessary-type-assertion -- to allow ts version bump
```
To fix this we need to remove `as Dependency[] ` (and also the import since it will no longer be used)